tethysapp/gwdm/public/js/edit_wells.js

```diff
@@ -75,15 +75,15 @@ var LIBRARY_OBJECT = (function() {
     });
 
     var viewIcon = function(cell, formatterParams){ //plain text value
-        return "<span class='glyphicon glyphicon-sunglasses view-region-tabulator'></span>";
+        return "<span class='sunglasses view-region-tabulator'></span>";
```

Comment:
Need to change `<span>` to `<i>` and add `bi` to classes:

e.g.:

```
return "<i class='bi bi-sunglasses view-region-tabulator'></i>";
```

Fix all instances of this in this file.